tests/e2e/wdio.conf.js

```diff
@@ -17,6 +17,16 @@ let testTile;
 const DEBUG = process.env.DEBUG;
 const DEFAULT_TIMEOUT = 2 * 60 * 1000;
 const DEBUG_TIMEOUT = 10 * 60 * 1000; //timeout in debug mode, allows more interaction with browser after test
+const CHROME_VERSION = process.env.CHROME_VERSION;
+const CHROME_OPTIONS_ARGS_DEBUG = [
+  'disable-gpu',
+  'deny-permission-prompts',
+  'ignore-certificate-errors',
+  'no-sandbox',
```

Comment:
Is this necessary? We didn't need it before and it means we're further away from real-world settings. I couldn't find good docs on why it would be necessary for us...

Reply:
I faced a problem while working with the default configuration and using Chromedriver instead of Devtools. The tests wouldn't start and kept showing `data:, ` instead of redirecting to `baseUrl` as expected. Thanks for pointing out this. I re-ran the standard configuration tests without the `no-sandbox` option, and they worked perfectly fine.

Reply:
The error I mentioned above appears on the [CI](https://github.com/medic/cht-core/actions/runs/6723769390/job/18276229947?pr=8663), but on my local machine, everything is working correctly.

`ERROR webdriver: session not created: session not created from tab crashed`

Reply:
I will investigate further how to fix this without adding the extra option. [This](https://stackoverflow.com/questions/59186984/selenium-common-exceptions-sessionnotcreatedexception-message-session-not-crea) is where I read about the `no-sandbox` option.

Reply:
I couldn't fix this without adding a `no-sandbox `option to arguments. To minimize impact, I only added when running tests with Chrome 90.